src/pages/Defi/components/EligibleCarousel.tsx

```diff
@@ -39,21 +40,33 @@ export const EligibleCarousel: React.FC<EligibleCarouselProps> = props => {
   const eligibleOpportunities = useAppSelector(selectAggregatedEarnUserStakingEligibleOpportunities)
   const filteredEligibleOpportunities = useMemo(() => {
     // opportunities with 1% APY or more
-    const filteredEligibleOpportunities = eligibleOpportunities
+    let filteredEligibleOpportunities = eligibleOpportunities
       .filter(o => bnOrZero(o.tvl).gt(50000) && bnOrZero(o.apy).gte(0.01))
-      .sort((a, b) => bn(b.apy).minus(a.apy).toNumber())
+      .sort((a, b) =>
+        bn(b.apy ?? '0')
+          .minus(a.apy ?? '0')
+          .toNumber(),
+      )
       .slice(0, 5)
 
     const foxFarmingV9 = eligibleOpportunities.find(
       eligibleOpportunity =>
         eligibleOpportunity.contractAddress === ETH_FOX_STAKING_CONTRACT_ADDRESS_V9,
     )
 
+    const rfoxOpportunity = eligibleOpportunities.find(
+      eligibleOpportunity => eligibleOpportunity.provider === DefiProvider.rFOX,
+    )
+
+    if (rfoxOpportunity) {
+      filteredEligibleOpportunities = [rfoxOpportunity, ...filteredEligibleOpportunities]
+    }
+
     if (!foxFarmingV9) {
       return filteredEligibleOpportunities
     }
 
-    // TEMP: Hardcode the Fox Farming V9 opportunity to be the first card until enough TVL is in the pool
+    // TEMP: Hardcode the rFOX opportunity to be the first card
```

Comment:
q: is this comment/naming still relevant given the line below?

Reply:
It is, the first item of filteredEigibleOpportunities is the rFOX opportunity

Reply:
The line below doesn't explicitly add rFOX as the first card anymore - what about:

```typescript
    if (!foxFarmingV9 || !rfoxOpportunity) {
      return filteredEligibleOpportunities
    }

    // TEMP: Hardcode the rFOX opportunity to be the first card
    const filteredEligibleOpportunitiesWithFoxFarmingV9 = uniqBy(
      [
        filteredEligibleOpportunities[0],
        rfoxOpportunity,
        foxFarmingV9,
        ...filteredEligibleOpportunities.slice(1),
      ].filter(isSome),
      'contractAddress',
    ).slice(0, 5)
```

WDYT?